core-bundle/src/Resources/contao/library/Contao/Config.php

```diff
@@ -105,6 +105,27 @@ class Config
 		'jpgQuality'      => 'contao.image.imagine_options[jpeg_quality]',
 	);
 
+	private static $arrToBeRemoved = array
+	(
+		'os'                    => true,
+		'browser'               => true,
+		'dbCharset'             => true,
+		'dbCollation'           => true,
+		'disableRefererCheck'   => true,
```

Comment:
We should provide a bundle config for this first, before deprecating it.

Reply:
I don’t know. It can be disabled by either setting the route attribute `_token_check=false` or by removing the listener. `disableRefererCheck` is already deprecated since Contao 4.7.3 AFAIK.

Reply:
Well, a bundle config is much easer than implementing a compiler class and manager plugin that removes the listener from the container. I think this setting is actually used a lot.